server/intelligence-service/app/main.py

```diff
@@ -10,18 +9,15 @@
     contact={"name": "Felix T.J. Dietrich", "email": "felixtj.dietrich@tum.de"},
 )
 
-# Global dictionary to store conversation states
-conversations: Dict[str, dict] = {}
-
 
 class ChatRequest(BaseModel):
-    message: str
-    thread_id: Optional[str] = None
+    session_id: str
+    message_content: str
 
 
 class ChatResponse(BaseModel):
-    response: str
-    thread_id: Optional[str] = None
+    session_id: str
+    message_content: str
 
 
 @app.post(
```

Comment:
Just move this endpoint to a separate file and let the send message code live in the body. I think this makes it a bit more complicated than it needs to be. We don't have to do controller + service for the intelligence service IMO just keep the code together for now